docs/versioned_docs/version-2.x/components.mdx

```diff
@@ -406,6 +415,10 @@ word vectors in your pipeline.
 
   ### ConveRTTokenizer
 
+:::caution Deprecated in 2.1
+The `ConveRTTokenizer` is deprecated and will be removed in 3.0. The [ConveRTFeaturizer](./components.mdx#convertfeaturizer)
```

Comment:
I was confused by following this once before (also Maxime lmk if you'd prefer to get the minors merged before doing this level of review)

I would prefer to just say it's deprecated. Because now you go to the featurizer, and see the caution at the bottom of that that you need to host the URL yourself since the public URL is taken down.

Reply:
Is it fine with you if we keep a list of things we want to improve after we've done all the merges? Because otherwise it's going to be hard to keep track of things (and will be longer to make progress on the PRs).

Reply:
yes totally!